prow/cmd/gerrit/README.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+[Gerrit] is a free, web-based team code collaboration tool.
+
+## Gerrit component
+
+The gerrit component is a Prow adapter for handling CI on gerrit workflows. It can poll gerrit
+changes from multiple gerrit instances, and trigger presubmits on Prow upon new patchsets on
+gerrit changes.
+
+## Underlying infra
+
+### Client
+
+We have a [gerrit-client package](/prow/gerrit/client) that provides a thin wrapper around  
+[andygrunwald/go-gerrit](https://github.com/andygrunwald/go-gerrit), which is a go client library
+for accessing the [Gerrit Code Review REST API](https://gerrit-review.googlesource.com/Documentation/rest-api.html)
+
+You can create a client instance by pass in a map of instance-name:project-ids, and pass in an oauth token path to
+start the client, like:
+
+```go
+projects := map[string][]string{
+	"foo.googlesource.com": {
+		"project-bar",
+		"project-baz",
+	},
+}
+
+c, err := gerrit.NewClient(projects)
+if err != nil {
+	// handle error
+}
+c.Start(cookiefilePath)
+```
+
+The client will try to refetch token from the path every 10 minutes.
+
+You should also utilize [grandmatriarch] to generate a token from a
+passed-in service account credential.
+
+If you need extra features, feel free to introduce new gerrit API functions to the client package.
+
+
+### Adapter
+
+The adapter package implements a controller that is periodically polling gerrit, and triggering
+presubmit jobs base on your prow config.
```

Comment:
This sounds almost identical to the description under `Gerrit Component`.

Reply:
separated into two docs now :-)